pkg/virtctl/vmexport/vmexport.go

```diff
@@ -791,6 +825,9 @@ func handleCreateFlags() error {
 	if keepVme {
 		return fmt.Errorf(ErrIncompatibleFlag, KEEP_FLAG, CREATE)
 	}
+	if portForward != "" {
```

Comment:
we should verify that the value is an actual int somewhere.

Reply:
Right, will do.